Makefile

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2021 light-river, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Is this your company? I'm not quite sure how we want to handle copywrites here but I will confer with @tasn and get back to you, we will probably need to setup a CLA.

Reply:
Yes it is - [tl;dr](https://tldrlegal.com/license/apache-license-2.0-%28apache-2.0%29) you have the right to use Apache licensed software commercially and privately.

Reply:
This was mostly copy-paste from private repo of which it's prepended to all files - you can have the code, devoid of license if you'd really like tho

Reply:
I see no problem with this file being Apache.